### PR TITLE
Find and compare

### PR DESCRIPTION
**Pull request by IMlame**
created route that finds nearby restaurants and orders them by their comparability score

### CODE REVIEW

File: routes/routes.py [hunk 1]
```diff
@@ -46,7 +47,24 @@ def compare_route():
...
-        return RestaurantComparer().compare_restaurants(rest1=rest1, rest2=rest2)
+        return str(RestaurantComparer().compare_restaurants(rest1=rest1, rest2=rest2))
 
     else:
         return "errorMessage: One or more restaurant IDs are missing"
+
+
+# receives longitude and latitude, as well as restaurant id
+# returns list of restaurants, from most compatible to least
+@app.route("/restaurants/search_and_find", methods=["GET"])
```
Comment (bryanesong) [inline routes/routes.py]: So this method takes in a restaurant and its location and returns similar restaurants. What is the search radius? Is it defaulted? What restaurants are being taken off of this restaurant? How many restaurants is the input restaurant being compared against?

The route name should also match the function name:
/restaurants/search_and_find and def compare_find_route should be named  more similarly  like def search_and_find_route

Also you may want to rethink the naming scheme for the API route. At the very least change the underscores to dashes. 

Some suggestions
From:
- /restaurants/search_and_find
To:
- /restaurants/nearby
- /restaurants/compare


https://restfulapi.net/resource-naming/

**Review by bryanesong — CHANGES_REQUESTED** [review]
need small changes to re-design route

**Comment by IMlame** [comment]
As far as I could tell, there is no way to specify the search radius of nearby restaurants through the yelp api.

**Review by bryanesong — APPROVED** [review]
lgtm!

**Pull request merged**